appveyor.yml

```diff
@@ -6,7 +6,7 @@ install:
   - "set PATH=%APPDATA%\\npm;%PATH%"
   - "npm install --no-optional"
 cache:
-  - "node_modules"
+  - "node_modules -> package.json"
```

Comment:
Just a note: Interesting syntax for cache wipe 🙂

Reply:
Yup, didn't know about it :) AppVeyor has some pretty cool features.